src/components/FarmDualCard/FarmDualCardDetails.tsx

```diff
@@ -403,11 +393,12 @@ const FarmDualCardDetails: React.FC<{
               </Box>
             </Box>
             <Box className={classes.inputVal} mb={2} mt={2} p={2}>
-              <input
+              <NumericalInput
                 placeholder='0.00'
                 value={stakeAmount}
-                onChange={(evt: any) => {
-                  setStakeAmount(evt.target.value);
+                fontSize={16}
+                onUserInput={(value) => {
```

Comment:
is onUserInput the equivalent to onChange?

Reply:
onUserInput is equivalent to onChange but it returns the typed string as parameter